src/codeview.jl

```diff
@@ -1,12 +1,20 @@
 const asm_syntax = Ref(:att)
 
+function highlight(io, lexer, x)
+    Sys.which("pygmentize") === nothing && return print(io, x)
+    open(pipeline(`pygmentize -f terminal256 -O style=lovelace -l $lexer`;
```

Comment:
We should probably detect if the terminal is able to handle that/have an option to turn it off

Reply:
Do you know if there is a cross platform way to do this?  I can use `tput colors` in Linux but it also has an issue when used in `tmux` https://unix.stackexchange.com/q/118806. `REPL` has `REPL.GlobalOptions.hascolor` but it's just a boolean value.

I think the simplest solution would be to make it configurable.  It also avoids hard-coding color scheme `-O style=lovelace`.  I chose it because I found that it works nicely with dark and light background.  But other users may have other preference.

Reply:
Actually, it looks like `pygmentize`'s default setup is OK.  It has some ad-hoc mechanism to detect the color setting (check if `$TERM` has `256` as a substring) https://bitbucket.org/birkenfeld/pygments-main/src/e63f7b68898445836091b51cfa81bf0613fb05eb/pygments/cmdline.py#lines-465:468

Reply:
Okay that sounds good.